CHANGELOG.md

```diff
@@ -54,6 +54,9 @@
 - Command Generate - Remove extra newline at the end of model file on mongoose.
 - Technical - Fix tests on dumper.
 
+### Fixed
+- Command Generate - Generate MongoDB hasMany.
```

Comment:
To fix, this change should be under "Added" section of "[Unreleased]".

Reply:
I though I would wait for the merge into devel to set the Changelog correctly, as it might change again